apps/pythonapi/examples/annotation_example.py

```diff
@@ -0,0 +1,115 @@
+# %% [md]
+#
+# # Annotations
+#
+# You can annotate your renderings using Vapor's build in annotations as well as MatPlotLib
+#
+# %%
+import example_utils
+from vapor import session, renderer, dataset, camera, utils
+
+ses = session.Session()
+data = example_utils.OpenExampleDataset(ses)
+
+ren = data.NewRenderer(renderer.TwoDDataRenderer)
+ren.SetVariableName(data.GetDataVarNames(2)[1])
+ses.GetCamera().ViewAll()
+
+# %% [md]
+#
+# ## Vapor Colorbars
+#
+# %%
+# Show the basic colorbar annotation for a renderer
+colorbar = ren.GetColorbarAnnotation()
+colorbar.SetEnabled(True)
+ses.Show()
+
+# %%
+# Customized colorbar annotation
+colorbar.SetSize((0.2, 0.25))
+colorbar.SetTitle("Colorbar Title")
+colorbar.SetCornerPosition((0.95, 0.95))
+ses.Show()
+
+# %% [md]
+#
+# ## MatPlotLib Colorbars
+#
+# You can customize the histograms as you would MatPlotLib's `pylab.colorbar`
```

Comment:
Are all of pyplot.colorbar's arguments/keywords available?

Reply:
Yes, they are forwarded to the PyPlot function.

Reply:
Awesome. Can you make a comment to that effect so that users know they can consult the `pylab.colobar` documentation for more informaiton?

Reply:
The documentation for the referenced function specifies that:

```
def ShowMatPlotLibColorbar(self, axes="auto", figsize=(9, 1.5), **kwargs)

         Shows a colorbar for Vapor's transfer function using pylab.colorbar.
         This function takes the same kwargs as pylab.colorbar and those
         parameters are passed onto the matplotlib function.
```

I can also add that as a comment here if you’d like, however, to explain my philosophy for the examples as a whole, I tried to create general high-level walkthroughs in the example notebooks and leave the specific details in the API documentation rather than duplicating it in the notebooks.